05.wc/lib/wc.rb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def output
+  opt = OptionParser.new
+  options = { lines: false, words: false, bytes: false }
+  opt.on('-l') { |v| options[:lines] = v }
+  opt.on('-w') { |v| options[:words] = v }
+  opt.on('-c') { |v| options[:bytes] = v }
+  opt.parse!(ARGV)
+  options.transform_values! { true } if options.values.none?
```

Comment:
```suggestion
  options = hogehoge
```

optionsもメソッドで取得するようにするとoutputメソッド全体がスッキリしそうです

Reply:
確かにそうですね！`parse_options`としてメソッド切り出ししました。